docs/deployment_ops_guide.md

```diff
@@ -0,0 +1,850 @@
+                                              (process)       (process)
+```
+
+A visualization of the confidence cache during a reorg of depth=2:
+
+```
+
+  *unshift*    *shift*    *shift*  *unshift*  *unshift*  *unshift*
+     │  │       ▲  ▲       ▲  ▲       │  │       │  │       │  │
+   ┌─▼──▼─┐   ┌─│──│─┐   ┌─│──│─┐   ┌─│──│─┐   ┌─▼──▼─┐   ┌─▼──▼─┐
+   │ ts10 │   │      │   │ │  │ │   │ │  │ │   │ ts10'│   │ ts11'│
+   ├──────┤   ├──────┤   ├─│──│─┤   ├─▼──▼─┤   ├──────┤   ├──────┤
+   │ ts09 │   │ ts09 │   │      │   │ ts09'│   │ ts09'│   │ ts10'│
+   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤
+   │ ts08 │   │ ts08 │   │ ts08 │   │ ts08 │   │ ts08 │   │ ts09'│
+   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤
+   │ ...  │ > │ ...  │ > │ ...  │ > │ ...  │ > │ ...  │ > │ ...  │
+   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤
+   │ ts02 │   │ ts02 │   │ ts02 │   │ ts02 │   │ ts02 │   │ ts03 │
+   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤
+   │ ts01 │   │ ts01 │   │ ts01 │   │ ts01 │   │ ts01 │   │ ts02 │
+   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤   ├──────┤
+   │ ts00 │   │ ts00 │   │ ts00 │   │ ts00 │   │ ts00 │   │ ts01 │
+   └──────┘   └──────┘   └──────┘   └──────┘   └──────┘   └─│──│─┘
+                                                            ▼  ▼  *pop*
+               reorg                            reorg     ┌──────┐
+               occurs                          resolves   │ ts00 │
+                here                             here     └──────┘
+                                                          (process)
+```
+
+> _Note: A large confidence protects you from large reorgs but causes a longer delay between startup and processing Tasks on a fully synced Lily node._
+
+> _Note: A small (or zero) confidence will allow tipsets which are reorged to be persisted despite only appearing on-chain for a brief time. This may be useful when attempting to analyze differences in state during reorgs._
+
+#### option: Task Timeout Window
+
+- arg: `lily [walk|watch] --window`
+- default: `30s`
+
+Configure a custom duration in which Lily does as much task processing as possible, any task(s) not completed within the window will be marked as incomplete. This means some epochs may not contain all task data. (And are candidates for later re-processing via gap find & fill.) Each `walk|watch` Job manages its own `window` value. This value is provided as [a parseable Golang duration](https://pkg.go.dev/time#ParseDuration).
+
+Example:
+
+```
+# passed as arg
+$ lily watch --window=60s
+```
+
+#### option: Persistence Config
+
+- config: `Storage.Postgresql|File.[Name]` (object)
+
+Lily can deliver scraped data to multiple PostgreSQL and File destinations on a per-Task basis. Each destination should be enumerated with a unique `[Name]` which will be used as an argument when starting a Task.
```

Comment:
@woshidama323 Here's where I attempt to call the `--storage` detail out and how I might make it clearer. Any further suggestions?

```suggestion
Lily can deliver scraped data to multiple PostgreSQL and File destinations on a per-Task basis. Each destination should be enumerated with a unique `[Name]` which will be used as an argument (via `--storage`) when starting a Task. (See [config: Task Storage destination (TODO: linkneeded)](#linkneeded) for more information about assigning storage config to a task.)
```